server/internal/heavy/metrics.go

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright 2019 Insolar Technologies GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package heavy
+
+import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+)
+
+var (
+	statBadgerStartTime = stats.Float64(
```

Comment:
is it first metrics of heavy node?   There is a finalisation delta somwhere in code, why do we need new file?

Reply:
hm, first in heavy package, we can add it to executor package and make it public, but I think it will be less obvious